src/components/30-organisms/footer/index.js

```diff
@@ -82,16 +162,17 @@ class AXAFooter extends LitElement {
       link.addEventListener('click', this._handleLinkClick);
     });
 
+    if (this.firstRender) {
```

Comment:
rename to `slotsPrepared`

Reply:
done with `slotsNotPrepared` because of it inverted boolean logic